server/handler.go

```diff
@@ -0,0 +1,59 @@
+package server
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/flashbots/go-template/util"
+)
+
+func (s *Server) handleAPI(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+}
+
+func (s *Server) handleHealthcheck(w http.ResponseWriter, r *http.Request) {
+	s.isHealthyMx.RLock()
+	defer s.isHealthyMx.RUnlock()
+
+	if !s.isHealthy {
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
should this not rather return [503 Service Unavailable](https://www.webfx.com/web-development/glossary/http-status-codes/what-is-a-503-status-code/)